test/flows/find_coronavirus_support_flow_test.rb

```diff
@@ -0,0 +1,597 @@
+require "test_helper"
+require "support/flow_test_helper"
+
+class FindCoronavirusSupportFlowFlowTest < ActiveSupport::TestCase
+  include FlowTestHelper
+
+  setup { testing_flow FindCoronavirusSupportFlow }
+
+  should "render a start page" do
+    assert_rendered_start_page
+  end
+
+  context "question: need_help_with" do
+    setup { testing_node :need_help_with }
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      should "have a next node of nation for an empty response" do
+        assert_next_node :nation, for_response: %w[none]
+      end
+
+      should "have a next node based on the response" do
+        assert_next_node :self_employed, for_response: %w[being_unemployed mental_health]
+      end
+    end
+  end
+
+  context "question: feel_unsafe" do
+    setup do
+      testing_node :feel_unsafe
+      add_responses need_help_with: %w[feeling_unsafe]
+    end
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    %w[yes concerned_about_others no not_sure].each do |response|
```

Comment:
I'm not sure what the convention is. Both the `afford_food` question and `feel_unsafe` are part of a conditional in the calculator. So, should I leave this loop in, or take them both out?

Reply:
Sure, the convention is to exercise the logic of the flow. So if the calculator has logic but the flow doesn't the flow tests don't need to test that logic as that should be covered by calculator unit tests.

I'd take all the loops out that don't involve any logic in the flow. The only question that needs more than one test for the next node in this is self employed since it has a conditional: https://github.com/alphagov/smart-answers/blob/396f93306c6ad512925851557265a22ae79042e1/app/flows/find_coronavirus_support_flow.rb#L116-L120